src/hypothesis/database.py

```diff
@@ -259,6 +266,17 @@ def save(self, key, value):
                 os.unlink(tmpname)
             assert not os.path.exists(tmpname)
 
+    def move(self, key1, key2, value):
```

Comment:
Minor: it wasn’t immediately obvious to me whether you were moving `key1` into `key2`, or vice versa.

Maybe `src_key` and `dst_key` for clarity?